docs/governance/decred-constitution.md

```diff
@@ -1,57 +1,42 @@
-# <img class="dcr-icon" src="/img/dcr-icons/Constitution.svg" /> Decred Constitution
+# Decred Constitution
 
-!!! note
-    The constitution was written prior to Decred's launch; some portions of it are no longer relevant and some are yet to be realized. Politeia will be used, when it launches, to propose amendments to the constitution for ratification by Voters.
+Decred (/ˈdi:ˈkred/, /dɪˈkred/, dee-cred) is a secure, adaptable and self-funding cryptocurrency with a system of community-based governance integrated into its blockchain. The project mission is to develop technology for the public benefit, with a primary focus on cryptocurrency technology. This constitution defines a set of principles which guide the decision-making of the project's stakeholders, and describes the processes through which the blockchain and Treasury are governed. This document is meant to manage the expectations of prospective and actual Decred users, so that they more clearly understand the social contract between themselves and the project. This document is not intended to take precedence over the Decred network's consensus rules, should any conflict arise.
 
-*Decred* (/ˈdi:ˈkred/, /dɪˈkred/, dee-cred) is an open, progressive, and self-funding cryptocurrency with a system of community-based governance integrated into its blockchain. The project mission is to develop technology for the public benefit, with a primary focus on cryptocurrency technology. *Decred*, as a currency and as a project, is bound by the following set of rules, which include guiding principles, a system of governance, and a funding mechanism. These rules have been established in an effort to create an equitable and sustainable framework within which to achieve *Decred*'s goals.
-
----
+------
```

Comment:
Change horizontal rule back to --- to minimize diff and increase consistency, or remove it alltogether.